util/signal/signal_posix.go

```diff
@@ -15,12 +15,14 @@
 package signal
 
 import (
+	"log"
```

Comment:
useless import?

Reply:
emm... If the logger package is down, sys log will work anyway.